main.go

```diff
@@ -1,12 +1,33 @@
 package main
 
+import "errors"
+
 func Divine()  {
 	
 }
 
 func Add2()  {
 	
 }
+
+func returnErr() error {
+	return errors.New("err1")
+}
+
+func Add3()  {
+	err:=returnErr()
+	println(err.Error())
+
+	a:=1
+	if a==1 {
+		println("a1")
+	}
+
+	if 1==1 {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA4000: identical expressions on the left and right side of the '==' operator (staticcheck)